content/blog/20241126-cache01.md

```diff
@@ -0,0 +1,160 @@
+---
+title: "Cache me if you can #1: An Intro to What Caches Are and Why You Need One"
```

Comment:
Zvaž vylepšení názvu souboru, použije se jako část URL...

Reply:
myslím tím tohle: `20241126-cache01.md`

Reply:
Teraz by sa to malo volať 20241126-cache-1-introduction.md